multiqc/modules/gatk/base_recalibrator.py

```diff
@@ -137,6 +140,61 @@ def add_quality_score_vs_no_of_observations_section(self):
             plot = plot,
         )
 
+    def add_reported_vs_empirical_section(self):
+        sample_data = []
+        data_labels = []
+
+        # Loop through the different data types
+        for rt_type_name, rt_type, in recal_table_type._asdict().items():
+            # This table appears to be the correct one to use for reported vs empirical
+            # https://github.com/broadinstitute/gatk/blob/853b53ec2a3ac2d90d7d82a6c8451e29a34692d2/src/main/resources/org/broadinstitute/hellbender/utils/recalibration/BQSR.R#L148
+            sample_tables = self.gatk_base_recalibrator[rt_type]['recal_table_1']
+            if len(sample_tables) == 0:
+                continue
+
+            reported_empirical = {}
+            for sample, table in sample_tables.items():
+                reported_empirical[sample] = []
+                table_rows = [dict(zip(table, r)) for r in zip(*table.values())]
+                table_rows.sort(key=lambda r: r['QualityScore'])
+                for reported, group in groupby(table_rows, lambda r: r['QualityScore']):
+                    g = list(group)
+                    try:
+                        reported_empirical[sample].append(
+                            {
+                                'x': int(reported),
+                                'y': sum(float(r['EmpiricalQuality']) for r in g) / len(g),
+                            }
+                        )
+                    except ZeroDivisionError:
+                        reported_empirical[sample][int(reported)] = 0
+                        reported_empirical[sample].append({'x': int(reported), 'y': 0})
```

Comment:
I know in Python were supposed to "ask for forgiveness", but I think this is clearer.
```suggestion
                    reported_empirical[sample].append(
                        {
                            'x': int(reported),
                            'y': sum(float(r['EmpiricalQuality']) for r in g) / len(g) if len(g) > 0 else 0,
                        }
                    )
```